artemis-commons/src/main/java/org/apache/activemq/artemis/utils/ByteUtil.java

```diff
@@ -39,9 +39,9 @@ public class ByteUtil {
    private static final String prefix = "^\\s*(\\d+)\\s*";
    private static final String suffix = "(b)?\\s*$";
    private static final Pattern ONE = Pattern.compile(prefix + suffix, Pattern.CASE_INSENSITIVE);
-   private static final Pattern KILO = Pattern.compile(prefix + "k" + suffix, Pattern.CASE_INSENSITIVE);
-   private static final Pattern MEGA = Pattern.compile(prefix + "m" + suffix, Pattern.CASE_INSENSITIVE);
-   private static final Pattern GIGA = Pattern.compile(prefix + "g" + suffix, Pattern.CASE_INSENSITIVE);
+   private static final Pattern KILO = Pattern.compile(prefix + "ki?" + suffix, Pattern.CASE_INSENSITIVE);
```

Comment:
This will break anyone using existing styles. Add new style support but don't break existing

Reply:
Michael, that was strictly my intention. the unit test that I added verifies that. how does this break anything?

Reply:
@michaelandrepearce I didn't see him changing the style.. just adding  a new one... unless I missed something.